mep/books/models.py

```diff
@@ -318,6 +323,11 @@ def translators(self):
         '''work creators with type translator'''
         return self.creator_by_type('Translator')
 
+    @property
+    def illustrators(self):
```

Comment:
Is this actually used anywhere ?

Reply:
IDE says no, and honestly I don't even remember adding this...in any case, not necessary now that we have the new detail page logic. I'll remove it

Reply:
Seems this also applies to `editors()` and `translators()`. should I remove those as well?

Reply:
I think those are used somewhere still - maybe in tests if nowhere else.

Reply:
yeah, looks like just tests. I'll leave them alone.